plugins/confluent/credentials.go

```diff
@@ -0,0 +1,91 @@
+package confluent
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func CloudCredentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.Credentials,
```

Comment:
Looking at the docs, I see that the Confluent CLI also supports API keys. To avoid confusion with that and to keep forwards compatibility for if we'd ever want to support these API keys in the future, it would be best to pick a less generic name for this username/password credential type.

For example:

```suggestion
		Name:          credname.UserLogin,
```

(Note: this one would need to be added to `sdk/schema/credname/names.go` first)